File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -3695,4 +3695,298 @@ loadedDocument.Close(true);
+        Next
+
+        Dim isEmbbed As Boolean = signerCertificate.CrlCertificate.IsEmbedded
+        Dim validForm As DateTime = signerCertificate.CrlCertificate.ValidFrom
+        Dim validTo As DateTime = signerCertificate.CrlCertificate.ValidTo
+    End If
+Next
+'Close the document.
+loadedDocument.Close(True)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Retrieve signed revision information from digital signature
+
+The following code example illustrates how to retrieve signed revision information from digital signature using [Revision](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html#Syncfusion_Pdf_Parsing_PdfLoadedSignatureField_Revision) property of [PdfLoadedSignatureField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html). 
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Load an existing PDF document.
+FileStream inputStream = new FileStream(@"Input.pdf", FileMode.Open, FileAccess.Read);
+PdfLoadedDocument document = new PdfLoadedDocument(inputStream);
+//Get the document revisions. 
+PdfRevision[] revisions = document.Revisions;
+foreach (PdfRevision rev in revisions)
+{
+    //Get revision start position.
+    long startPosition = rev.StartPosition;
+}
+//Load the existing signature field.
+PdfLoadedSignatureField field = document.Form.Fields[0] as PdfLoadedSignatureField;
+//Get the revision of the signature.
+int revisionIndex = field.Revision;
+//Close the document.
+document.Close(true);
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+//Load an existing PDF document
+PdfLoadedDocument document = new PdfLoadedDocument("Input.pdf");
+//Get the document revisions. 
+PdfRevision[] revisions = document.Revisions;
+foreach (PdfRevision rev in revisions)
+{
+    //Get revision start position.
+    long startPosition = rev.StartPosition;
+}
+//Load the existing signature field.
+PdfLoadedSignatureField field = document.Form.Fields[0] as PdfLoadedSignatureField;
+//Get the revision of the signature.
+int revisionIndex = field.Revision;
+//Close the document.
+document.Close(true);
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+'Load an existing PDF document.
+Dim document As PdfLoadedDocument = New PdfLoadedDocument("Input.pdf")
+'Get the document revisions. 
+Dim revisions As PdfRevision() = document.Revisions
+For Each rev As PdfRevision In revisions
+    'Get revision start position.
+    Dim startPosition As Long = rev.StartPosition
+Next
+'Load the existing signature field.
+Dim field As PdfLoadedSignatureField = TryCast(document.Form.Fields(0), PdfLoadedSignatureField)
+'Get the revision of the signature.
+Dim revisionIndex As Integer = field.Revision
+'Close the document.
+document.Close(True)
+
+{% endhighlight %}
+
+{% endtabs %}    
+
+## Extract images from PDF documents with better memory consumption and performance
```

Comment:
Can we move this topic under the Working with image extraction?

Reply:
Moved the topic to image extraction